doc/source/reference/sparse.linalg.svds-propack.rst

```diff
@@ -0,0 +1,8 @@
+.. _sparse.linalg.svds-propack:
+
+svds(solver='propack')
+----------------------------------------
+
+.. scipy-optimize:function:: scipy.sparse.linalg.svds
```

Comment:
A leftover?

Reply:
This is intentional. There is some code in `doc/source/scipyoptdoc.py` that we're also using for `svds` to render method-specific documentation; see gh-14251.

Reply:
oh I see, resolving then.